ppl/src/main/antlr/OpenDistroPPLParser.g4

```diff
@@ -77,8 +77,6 @@ evalCommand
 
 headCommand
     : HEAD
-    (KEEPLAST EQUAL keeplast=booleanLiteral)?
-    (WHILE LT_PRTHS whileExpr=logicalExpression RT_PRTHS)?
```

Comment:
remove `KEEPLAST` and `WHILE` token as well if no use?

Reply:
Done, thanks!